lua/entities/gmod_wire_expression2/core/console.lua

```diff
@@ -35,46 +35,74 @@ local function tokenizeAndGetCommands(str)
 	return commands
 end
 
-local function validConCmd(self, command)
+---@param cmd_var "cmd"|"var"
+---@return boolean override # Whether the user set their whitelist to "" (allow everything)
+---@return table whitelist # Whitelist for specific commands otherwise.
+local function getWhitelist(ply, cmd_var)
```

Comment:
Could you make the cmd_var instead be the whole command? Also could you make it return only a table and have the callers check if the table is empty for the 'override'.

Reply:
Done